web/Dockerfile

```
@@ -1,5 +1,5 @@
 # Fails to build styled-components with node 18
-FROM node:16 as base
+FROM node:14 as base

```

```suggestion
FROM node:16 as base
```
Why the downgrade? I believe there should be a good reasoning for this if we choose to go that way.

Please note that active support for Node v14 (LTS) ended October '21, and Security support ends April '23 (in 7 months).
In addition, active support for Node v16 (LTS) ends October '22 (soon™), and Security support ends September '23. As such, it could be worth considering a bump to Node 18 in the near future [?], or at least avoid using an already dated version.
Source: https://endoflife.date/nodejs

---

Found another solution, the problem is that styled components using yarn pnp crashes (not node 14), but found another solution to it: [styled components post-install causing noise in yarn pnp](https://github.com/equinor/template-fastapi-react/pull/61/commits/03dfac0d20ca0d82cffd04c000752642882b8188)